test/e2e/app-dir/next-script/next-script.test.ts

```diff
@@ -0,0 +1,26 @@
+import { nextTestSetup } from 'e2e-utils'
+import { BrowserInterface } from '../../../lib/browsers/base'
+
+describe('Script component with crossOrigin props', () => {
+  const { next } = nextTestSetup({
+    files: __dirname,
+  })
+
+  it('should be set crossOrigin also in preload link tag', async () => {
+    let browser: BrowserInterface
+
+    try {
+      browser = await next.browser('/')
+
+      expect(
+        await browser
+          .elementByCss(
+            'link[href="https://code.jquery.com/jquery-3.7.1.min.js"]'
+          )
+          .getAttribute('crossorigin')
+      ).toBe('use-credentials')
+    } finally {
+      if (browser) await browser.close()
+    }
```

Comment:
```suggestion
    const browser = await next.browser('/')
    const crossOrigin = await browser
      .elementByCss('link[href="https://code.jquery.com/jquery-3.7.1.min.js"]')
      .getAttribute('crossorigin')

    expect(crossOrigin).toBe('use-credentials')
```

Reply:
AFAIK you don't need manual close at the end!

Reply:
I think it make sense more. Thank you! I reflected.